### ISSUE TITLE
A bug in the model?

### ISSUE THREAD

**Issue by violet-zct** [thomasmesnard/DeepMind-Teaching-Machines-to-Read-and-Comprehend#5]
I found that when representing the context word (concatenating the forward and backward rnn), the backward rnn is not reversed to match the sequence order of the forward rnn.

**Comment by rabberbock** [comment]
@violet-zct Were you able to use the model to actually make predictions?

**Comment by violet-zct** [comment]
Yes, I have replied to one of the posts concerning this issue. Create your
test data stream and add another DataStreamMonitoring to the extensions in
train.py.

2016-05-24 3:03 GMT+08:00 rabberbock notifications@github.com:
